bindings/matrix-sdk-crypto-ffi/src/verification.rs

```diff
@@ -178,55 +423,166 @@ pub struct ConfirmVerificationResult {
+    }
+
+    /// Get the unique ID of this verification request
+    pub fn flow_id(&self) -> String {
+        self.inner.flow_id().as_str().to_owned()
+    }
+
+    /// Get the room id if the verification is happening inside a room.
+    pub fn room_id(&self) -> Option<String> {
+        self.inner.room_id().map(|r| r.to_string())
+    }
+
+    /// Has the verification flow that was started with this request finished.
+    pub fn is_done(&self) -> bool {
+        self.inner.is_done()
+    }
+
+    /// Is the verification request ready to start a verification flow.
+    pub fn is_ready(&self) -> bool {
+        self.inner.is_ready()
+    }
+
+    /// Did we initiate the verification request
+    pub fn we_started(&self) -> bool {
+        self.inner.we_started()
+    }
+
+    /// Has the verification request been answered by another device.
+    pub fn is_passive(&self) -> bool {
+        self.inner.is_passive()
+    }
+
+    /// Has the verification flow that been cancelled.
+    pub fn is_cancelled(&self) -> bool {
+        self.inner.is_cancelled()
+    }
+
+    /// Get info about the cancellation if the verification request has been
+    /// cancelled.
+    pub fn cancel_info(&self) -> Option<CancelInfo> {
+        self.inner.cancel_info().map(|v| v.into())
+    }
+
+    /// Get the supported verification methods of the other side.
+    ///
+    /// Will be present only if the other side requested the verification or if
+    /// we're in the ready state.
+    pub fn their_supported_methods(&self) -> Option<Vec<String>> {
+        self.inner.their_supported_methods().map(|m| m.iter().map(|m| m.to_string()).collect())
+    }
+
+    /// Get our own supported verification methods that we advertised.
+    ///
+    /// Will be present only we requested the verification or if we're in the
+    /// ready state.
+    pub fn our_supported_methods(&self) -> Option<Vec<String>> {
+        self.inner.our_supported_methods().map(|m| m.iter().map(|m| m.to_string()).collect())
+    }
+
+    /// Accept a verification requests that we share with the given user with
+    /// the given flow id.
+    ///
+    /// This will move the verification request into the ready state.
+    ///
+    /// # Arguments
+    ///
+    /// * `user_id` - The ID of the user for which we would like to accept the
+    /// verification requests.
+    ///
+    /// * `flow_id` - The ID that uniquely identifies the verification flow.
+    ///
+    /// * `methods` - A list of verification methods that we want to advertise
+    /// as supported.
+    pub fn accept(&self, methods: Vec<String>) -> Option<OutgoingVerificationRequest> {
+        let methods = methods.into_iter().map(VerificationMethod::from).collect();
+        self.inner.accept_with_methods(methods).map(|r| r.into())
+    }
+
+    /// Cancel a verification for the given user with the given flow id using
+    /// the given cancel code.
+    pub fn cancel(&self) -> Option<OutgoingVerificationRequest> {
```

Comment:
Isn't it missing the cancel code as param?

Reply:
I don't think so? If the user cancels, the code is `m.user`, if we need to cancel for any other reason the SDK cancels itself.